src/main/java/com/github/swapbook/email/EmailServiceImpl.java

```diff
@@ -0,0 +1,31 @@
+package com.github.swapbook.email;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.mail.SimpleMailMessage;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.stereotype.Component;
+
+@Component
+public class EmailServiceImpl implements EmailService {
+
+    private static final Logger log = LoggerFactory.getLogger(EmailServiceImpl.class);
+
+    @Autowired
+    private JavaMailSender emailSender;
+
+    public void sendSimpleMessage(String to, String subject, String text) {
+
+        SimpleMailMessage message = new SimpleMailMessage();
+
+        message.setTo(to);
+        message.setSubject(subject);
+        message.setText(text);
```

Comment:
czy to ma włączone formatowanie html ?

Reply:
nope, ten nie. Jeśli chcemy formatowanie to można by użyż znacznie bardziej robudowanego MimeMessage zamiast SimpleMailMessage

Czy zmienić?

Reply:
chcemy wysyłać link do aktywacji konta więc chyba by się przydało XD

Reply:
a okej, to dołożę

Reply:
no w plaintexcie jak dasz pełen tekst z https:// to automatycznie go poczta skonwertuje na link. Ale już dodane też jak coś